apps/chat/docker-compose.yml

```diff
@@ -5,15 +5,15 @@ services:
     user: postgres
     restart: always
     healthcheck:
-      test: 'pg_isready -U user --dbname=postgres'
+      test: 'pg_isready -U user --dbname=onechat'
```

Comment:
There was a diff between the database name in the `.env` you provided earlier and the actual setup, so I adjusted it to match the environment variables. @natew

Reply:
Since you are creating a Database onechat inside init.sql, i dont think you intended to make the onechat as an default DB. Sticking to postgres as default one is correct (or at least should have different name than onechat if you want to create onechat inside init.sql)

Reply:
makes sense @cermakondrej

Reply:
Oh yeah you are right, thank you! @cermakondrej